ts/packages/ts/src/ConfigSchema.ts

```diff
@@ -61,6 +61,195 @@ export abstract class ConfigSchema implements IConfigSchema {
           /^(?:(?=[^:\/]{1,253})(?!-)[a-zA-Z0-9-]{1,63}(?<!-)(?:\.(?!-)[a-zA-Z0-9-]{1,63}(?<!-))*(?::[0-9]{1,5})?\/)?((?![._-])(?:[a-z0-9._-]*)(?<![._-])(?:\/(?![._-])[a-z0-9._-]*(?<![._-]))*)(?::(?![.-])[a-zA-Z0-9_.-]{1,128})?$/gm.test(
             value
           ),
+        AwsRegion: ({ value }) => 
```

Comment:
Please use `Set().has()` instead of multiple OR statements. This change will decrease complexity to O(1) from O(n)
```javascript
AwsRegion: ({ value }) => new Set(["af-south-1", "ap-east-1"]).has(value)
```

Reply:
done